csbuild/toolchain/toolchain.py

```diff
@@ -804,7 +804,7 @@ def __getattribute__(self, name):
 									with Use(dummy):
 										return getattr(self, name)
 
-							if lastClass:
+							if lastClass: # pylint: disable=bad-option-value,no-else-raise
```

Comment:
`elif` -> `if` - and we should probably harden this with something like `if lastClass is not None` or whatever is appropriate here. That'll probably quiet the bad-option-value one.

Reply:
The `bad-option-value` is only for python 2 compatibility.  See my comment below.  But, we can fix the `no-else-raise` to get rid of both.